Rain Alert Mail Sender/main.py

```diff
@@ -0,0 +1,50 @@
+import requests
+from smtplib import SMTP
+
+#turn off certain security criteria in sender mail address
```

Comment:
What type of security criteria? Kindly mention them as doc string

Reply:
@kaustubhgupta done.